Resources/Prototypes/_Goobstation/Devil/clauses.yml

```diff
@@ -0,0 +1,229 @@
+# SPDX-FileCopyrightText: 2025 GoobBot <uristmchands@proton.me>
+# SPDX-FileCopyrightText: 2025 Solstice <solsticeofthewinter@gmail.com>
+# SPDX-FileCopyrightText: 2025 SolsticeOfTheWinter <solsticeofthewinter@gmail.com>
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+# The ID is what is searched for on the paper, coming after the target identifier.
+# Any spaces on the written portion will be removed, so keep it all lowercase and one word.
+
+- type: clause
+  id: soulownership
+  clauseWeight: 25
+  event: !type:DevilContractSoulOwnershipEvent
+
+- type: clause
+  id: weakness
+  clauseWeight: -35
+  damageModifierSet: DevilDealPositive
+
+# Disabled - Its used by shitters mostly
+# // VORE //
+#- type: clause
+#  id: avarice
+#  clauseWeight: -30
+#  addedComponents:
+#  - type: Devourer
+#    foodPreferenceWhitelist:
+#      components:
+#      - HumanoidAppearance
+#    stomachStorageWhitelist:
+#      components:
+#      - MobState
+#      - Item
+#    chemical: Ichor
+#    healRate: 7.5
+#    whitelist:
+#      components:
+#      - MobState
+#      - Door
+#      - Item
+#      tags:
+#      - Wall
+
+- type: clause
+  id: fearoffire
+  clauseWeight: -20
+  damageModifierSet: HellSpawn
+  addedComponents:
+  - type: SpecialHighTempImmunity
+
+- type: clause
+  id: brokeness
+  clauseWeight: -20
+  spawnedItems:
+  - ClothingShoesRealJordansAutographed
+
+- type: clause
+  id: fearofdeath
+  clauseWeight: -25
+  addedComponents:
+  - type: MobThresholds
+    thresholds:
+      0: Alive
+      200: Critical
+      201: Dead
```

Comment:
This doesn't seem to work, presumably because both parties already have this component and there's no special case for overriding existing components' values